doc/reference.md

```diff
@@ -896,19 +900,35 @@ constants.
 Supported brands for `card` payments accepted during the In-App Payments SDK checkout
 flow.
 
-* `VISA` - Visa Inc. credit or debit card.
-* `MASTERCARD` - Mastercard Incorporated credit or debit card.
-* `AMERICAN_EXPRESS` - American Express Company credit card.
-* `DISCOVER` - Discover Financial Services credit card.
-* `DISCOVER_DINERS` - Diners Club International credit card.
-* `JCB` - Japan Credit Bureau credit card.
-* `CHINA_UNION_PAY` - China UnionPay credit card.
-* `OTHER_BRAND` - An unexpected card type.
+* `visa` - Visa Inc. credit or debit card.
+* `mastercard` - Mastercard Incorporated credit or debit card.
+* `americanExpress` - American Express Company credit card.
+* `discover` - Discover Financial Services credit card.
+* `discoverDiners` - Diners Club International credit card.
+* `jCB` - Japan Credit Bureau credit card.
```

Comment:
How come the letters CB are capitalized and j is not?

Reply:
the convention is to start with lower case, rest should keep original case.